actix-server/src/accept.rs

```diff
@@ -121,9 +121,9 @@ impl Availability {
         } else {
             let shift = 1 << idx as u128;
 
-            debug_assert_ne!(self.0[offset] & shift, 0);
-
-            self.0[offset] ^= shift;
+            if self.0[offset] & shift != 0 {
+                self.0[offset] ^= shift;
+            }
```

Comment:
pure bitwise set to zero ?

```suggestion
            self.0[offset] & ~shift
```

Reply:
oh what rust syntax is ! for this ?

Reply:
yea. fixed.